src/main/java/com/internship/backend/model/NewDate.java

```diff
@@ -0,0 +1,45 @@
+package com.internship.backend.model;
+
+import jakarta.persistence.Embeddable;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Embeddable
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class NewDate {
```

Comment:
Could you chach if LocalDateTime class could be used instead of this one? If not, could some unit tests be added for the public methods?